distribution/std-lib/Base/src/Data/Text/Extensions.enso

```diff
@@ -37,15 +37,15 @@ Text.length =
    text-processing applications.
 Text.each : (Text -> Any) -> Nothing
 Text.each function =
-    iterator = BreakIterator.getCharacterInstance []
-    iterator.setText [this]
+    iterator = BreakIterator.getCharacterInstance
+    iterator.setText this
 
-    fst = iterator.first []
-    nxt = iterator.next []
+    fst = iterator.first
+    nxt = iterator.next
```

Comment:
Surely we can use `next` here now, given that we no longer have UFCS! Other locations too.

Reply:
We sure can. But this PR had too much boy-scouting in other places already, it's just too much for one person :P